Fix problem where setup.py cannot find README.rst in main directory
--- setup.py.back	2023-01-19 14:18:38.000000000 +0100
+++ setup.py	2023-01-19 14:21:12.000000000 +0100
@@ -7,7 +7,7 @@
     name='SpatialDE',
     version='1.1.3',
     description='Spatial and Temporal DE test',
-    long_description=(HERE.parent / 'README.rst').read_text(),
+    long_description=(HERE / 'README.rst').read_text(),
     url='https://github.com/Teichlab/SpatialDE',
     packages=find_packages(),
     include_package_data=True,
